src/Web/Telegram/API/Bot/API.hs

```diff
@@ -5,7 +5,10 @@
 
 module Web.Telegram.API.Bot.API
   ( -- * Functions
-    getMe
+    runClient
+  , runClientEx
```

Comment:
Direct IO actions may be convenient in little programs, but I'm not sure they are useful in complex cases.